docs/en/how-to/loading-fixtures.rst

```diff
@@ -106,3 +106,13 @@ then pass ``true`` to the 2nd argument of execute:
 .. code-block:: php
     <?php
     $executor->execute($loader->getFixtures(), append: true);
+
+By default the ``ORMExecutor`` will wrap the purge and the load of fixtures
+in a single transaction, which is the recommended way, but in some cases (for
+example if loading your fixtures is too slow and causes timeouts) you may
+want to wrap the purge and the load of every fixture in its own transaction.
+To do so, you can pass ``false`` to the third argument:
+
+.. code-block:: php
+    <?php
+    $executor->execute($loader->getFixtures(), singleTransaction: false);
```

Comment:
This does not look right. `$singleTransaction` is a constructor parameter of `ORMExecutor`, not a parameter of the `execute()` method.

Reply:
My bad! Thanks for the notice